docs/src/components/home/ProductSuite.tsx

```diff
@@ -44,11 +49,20 @@ function ProductSuite() {
   return (
     <Box
       ref={ref}
-      sx={{
-        bgcolor: (theme) => (theme.palette.mode === 'dark' ? 'primaryDark.900' : 'grey.50'),
+      sx={(theme) => ({
+        ...(!theme.vars
+          ? {
+              bgcolor: theme.palette.mode === 'dark' ? 'primaryDark.900' : 'grey.50',
+            }
+          : {
+              bgcolor: 'grey.50',
+              [theme.getColorSchemeSelector('dark')]: {
```

Comment:
Why not use the `theme.applyDarkStyles` as everywhere else?

Reply:
All fixed.